src/test/unit/algorithms/SpatialPoolerTest.cpp

```diff
@@ -1660,7 +1670,7 @@ TEST(SpatialPoolerTest, getOverlaps) {
   SpatialPooler sp;
   const vector<UInt> inputDim = {5};
   const vector<UInt> columnDim = {3};
-  sp.initialize(inputDim, columnDim);
+  setup(sp, inputDim, columnDim);
```

Comment:
this is example of use of setup. initialize() would throw assert, so we call setup() that overrides sparsity to 0.5f